Sources/Task.swift

```diff
@@ -88,19 +88,6 @@ extension Task: Hashable {
 	}
 }
 
-private func ==<Key, Value: Equatable>(lhs: [Key: Value]?, rhs: [Key: Value]?) -> Bool {
```

Comment:
This was removed because of auto inference of equality?

Reply:
`[Key: Value]?` is `Equatable` automatically now because `Optional` and `Dictionary` both have conditional `Equatable` conformances.